src/Toggle/ToggleChecker.php

```diff
@@ -51,9 +46,28 @@ public function isToggleActive( $toggleName )
         if ( !$this->toggleGateway->isToggleVisibleForUsers($toggleName) ) {
             return false;
         }
-        $groupPolicyResponse = $this->groupPolicy->getTogglePolicy($toggleName, $this->group);
+        $groupPolicyResponse = $this->transformByType(
+            $this->groupPolicy->getTogglePolicy($toggleName, $this->group),
+            $this->toggleGateway->getType($toggleName));
         return $groupPolicyResponse->isEnabled() ||
```

Comment:
Yes, group policy is stronger than user policy. What is wrong with that?
If toggle type is 'group' then user policy is irrelevant.